pal/ios-osx/tlsio_appleios.c

```diff
@@ -521,7 +521,14 @@ static void dowork_poll_socket(TLS_IO_INSTANCE* tls_io_instance)
     CFStreamCreatePairWithSocketToHost(NULL, tls_io_instance->hostname, tls_io_instance->port, &tls_io_instance->sockRead, &tls_io_instance->sockWrite);
     if (tls_io_instance->sockRead != NULL && tls_io_instance->sockWrite != NULL)
     {
-        if (CFReadStreamSetProperty(tls_io_instance->sockRead, kCFStreamPropertySSLSettings, kCFStreamSocketSecurityLevelNegotiatedSSL))
+        
+        CFStringRef keys[1] = {kCFStreamPropertySocketSecurityLevel};
+        CFStringRef values[1] = {kCFStreamSocketSecurityLevelNegotiatedSSL};
+
+        CFDictionaryRef test = CFDictionaryCreate(NULL , (void *)keys , (void *)values , 1,  NULL , NULL);
```

Comment:
reason why this is named test?

Reply:
No reason. change to tls_io_dictionary?

Reply:
Yea if this is going to be merged this should be updated. Is this PR testing a fix or looking to be merged. The title is a little confusing

Reply:
Updated and title corrected.